Adding code examples for Late Metrics Submission via DogStatsD. (#16567)
DataDog/documentation: content/en/metrics/custom_metrics/late_metrics.md

{{< /programming-lang-wrapper >}}

> **To submit late metrics via the Agent**, make sure you have Agent version 7.40.0 or later installed. This version includes an updated DogStatsD interface, which currently supports **Java** , **GoLang** and **.NET**. This allows you to send delayed metric points through the Agent.

Reviewer: ```suggestion
**To submit late metrics with the Agent**, make sure you have Agent version 7.40.0 or later installed. This version includes an updated DogStatsD interface, which supports **Java**, **GoLang**, and **.NET**. This allows you to send delayed metric points through the Agent.
```
